app/src/main/java/com/example/harvestfresh/fragments/EventsFragment.java

```diff
@@ -0,0 +1,232 @@
+package com.example.harvestfresh.fragments;
+
+import android.Manifest;
+import android.app.Dialog;
+import android.location.Location;
+import android.location.LocationRequest;
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+
+import android.os.Looper;
+import android.text.TextUtils;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Toast;
+
+import com.example.harvestfresh.R;
+import com.google.android.gms.common.ConnectionResult;
+import com.google.android.gms.common.ErrorDialogFragment;
+import com.google.android.gms.common.GooglePlayServicesUtil;
+import com.google.android.gms.location.LocationSettingsRequest;
+import com.google.android.gms.maps.CameraUpdate;
+import com.google.android.gms.maps.CameraUpdateFactory;
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.OnMapReadyCallback;
+import com.google.android.gms.maps.SupportMapFragment;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.MarkerOptions;
+import com.google.android.gms.tasks.OnFailureListener;
+import com.google.android.gms.tasks.OnSuccessListener;
+
+
+public class EventsFragment extends Fragment implements OnMapReadyCallback {
+
+    private final static String KEY_LOCATION = "location";
+    private final static int CONNECTION_FAILURE_RESOLUTION_REQUEST = 9000;
+
+    private long UPDATE_INTERVAL = 60000;
+    private long FASTEST_INTERVAL = 5000;
```

Comment:
Should these 2 also be static and final ?

Reply:
I had followed the formatting  found online so I was not sure since these are intervals for the MPAs API I but these two variables are not in the final committed code so it should be good.